description.md

```diff
@@ -1,37 +1,44 @@
-AppMap enhances the developer experience by analyzing runtime code behavior before you commit changes and go to production.
+AppMap is a runtime analysis and visualization platform. It analyzes your application's runtime code behavior before your changes go to production, and allows you to understand complex code interactions.
```

Comment:
We may want to focus on what AppMap for JetBrains does. We can get into the platform / CI features later (as you do).